starter/class/computer-player.js

```diff
@@ -0,0 +1,299 @@
+const Screen = require("./screen");
+// const TTT = require("./ttt");
+
+class ComputerPlayer {
+    static nextMove;
+    // Static method that checks all possible ways to win (X or O has reached 3)
+    static checkWin(grid) {
+        let total = 0;
+        // Score comparison to declare a winner
+        const declareWinner = function(X, O) {
+            if (X === 3) {
+                return "X";
+            } else if (O === 3) {
+                return "O";
+            } else {
+                return false;
+            }
```

Comment:
Simple tip here, you don't need an `else` here if your upper parts of the code are doing a `return`.  

So you can do:
```js
            if (X === 3) {
                return "X";
            } else if (O === 3) {
                return "O";
            }
            return false;
```

Reply:
Also, it would be nice if you had `declareWinner` as a function in ComputerPlayer, rather than belonging to checkWin.

But i notice that you have these methods here and in another class.  Maybe we can talk about that later

Reply:
Now as i think about it..
Since this function is getting fairly large, with internal methods you needed to help you check who won the game, you could create another class (let's call it "CheckGameWinner" class) so you can pull out all this `checkWin()` logic into.  That class will solely be responsible for being the judge/referee of the game.

In other words, you are "encapsulating" logic inside of a class specific to it, so that classes like ComputerPlayer doesn't have to worry about it.  

And anywhere you need to reference it, you can just call 

```js
  const winner = new CheckGameWinner(grid)
```


I'll let you figure out how to make a new class that does all this stuff